pgsql/pointcloud.sql.in

```diff
@@ -309,6 +309,10 @@ CREATE OR REPLACE FUNCTION PC_IsSorted(p pcpatch, attr text[], strict boolean de
 	RETURNS boolean AS 'MODULE_PATHNAME', 'pcpatch_is_sorted'
 	LANGUAGE 'c' IMMUTABLE STRICT;
 
+CREATE OR REPLACE FUNCTION PC_SetSchema(p pcpatch, pcid int4)
+	RETURNS pcpatch AS 'MODULE_PATHNAME', 'pcpatch_setschema'
+	LANGUAGE 'c' IMMUTABLE STRICT;
+
```

Comment:
Because it does more than just changing the `pcid` of a patch. It may change the data if the offsets/scales are different. In the future it may re-project the data if the srid is not the same. So after some discussion with @pblottiere we decided that `SetSchema` made a better name.